service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -50,19 +50,7 @@
 import greencity.exceptions.tariff.TariffAlreadyExistsException;
 import greencity.filters.TariffsInfoFilterCriteria;
 import greencity.filters.TariffsInfoSpecification;
-import greencity.repository.BagRepository;
-import greencity.repository.CourierRepository;
-import greencity.repository.OrderBagRepository;
-import greencity.repository.OrderRepository;
-import greencity.repository.DeactivateChosenEntityRepository;
-import greencity.repository.EmployeeRepository;
-import greencity.repository.LocationRepository;
-import greencity.repository.ReceivingStationRepository;
-import greencity.repository.RegionRepository;
-import greencity.repository.ServiceRepository;
-import greencity.repository.TariffLocationRepository;
-import greencity.repository.TariffsInfoRepository;
-import greencity.repository.UserRepository;
+import greencity.repository.*;
```

Comment:
We dont use star imports (correct me if wrong)

Reply:
Thanks. Improved imports